docs/dev_notes.rst

```diff
@@ -33,7 +35,7 @@ To test that an Astroconda package builds, you will need ``conda-build``::
    $ conda install conda-build
 
 #. Fork (if needed) and clone https://github.com/astroconda/astroconda-contrib
-#. Edit `poppy/meta.yaml <https://github.com/astroconda/astroconda-contrib/blob/master/poppy/meta.yaml>`_ to reflect the new ``version`` and ``git_tag``.
+#. Edit `poppy/meta.yaml <https://github.com/astroconda/astroconda-contrib/blob/develop/poppy/meta.yaml>`_ to reflect the new ``version`` and ``git_tag``.
```

Comment:
Based on what you mentioned in the webbpsf PR, this should probably be switched back to master